spec/components/post_revisor_spec.rb

```diff
@@ -485,10 +485,62 @@
     describe 'rate limiter' do
       fab!(:changed_by) { Fabricate(:coding_horror) }
 
+      before do
+        RateLimiter.enable
+        RateLimiter.clear_all!
+        SiteSetting.editing_grace_period = 0
+      end
+
+      after do
+        RateLimiter.disable
```

Comment:
I don't think we need this... looks like we are doing this needlessly in a bunch of places.

https://github.com/discourse/discourse/blob/90ab3b1c7575772aed87755e302aea530909af9a/spec/rails_helper.rb#L109-L109